ui_framework/components/icon/delete_icon.js

```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+import { KuiIcon } from './kui_icon';
+
+export function DeleteIcon() {
+  return <KuiIcon className="fa-trash"/>;
```

Comment:
_Nitpick_: As these become a whole lot of tiny files, I think I'd personally go for starting with adding them to `ui_framework/components/icon/index.js` like this:

```js
import React from 'react';
import { KuiIcon } from './kui_icon';

export const DeleteIcon = () => <KuiIcon className="fa-trash"/>

export const CreateIcon = () => <KuiIcon className="fa-plus"/>
```

etc, and in `ui_framework/components/index.js`:

```
export * from './icon';
```

then later on extract even further if needed. That would also make it easy to extract one icon into it's own file and do something like this:

```
export { UserIcon } from './user_icon'
```

instead of defining it in the same file.